cambridgeScript/parser/lexer.py

```diff
@@ -108,13 +119,14 @@ def __eq__(self, other):
             return True
         return super().__eq__(other)
 
+
 _TOKENS = [
     ("IGNORE", r"/\*.*\*/|(?://|#).*$|[ \t]+"),
     ("NEWLINE", r"\n"),
     ("KEYWORD", "|".join(Keyword)),
-    ("SYMBOL", r"-|(" + "|".join(re.escape(s) for s in Symbol if s != "-") + ")"),
-    ("LITERAL", r'-?[0-9]+(?:\.[0-9]+)?|".*?"'),
-    ("IDENTIFIER", r"[A-Za-z_][A-Za-z0-9_]*"),
+    ("LITERAL", r'-?[0-9]+(?:\.[0-9]+)?|".*?"|TRUE|FALSE'),
+    ("SYMBOL", r"(" + "|".join(map(re.escape, Symbol)) + ")"),
+    ("IDENTIFIER", r"[A-Za-z0-9]+"),
```

Comment:
```suggestion
    ("IDENTIFIER", r"[A-Za-z][A-Za-z0-9]*"),
```
Just to make sure literals can't start with a digit in case stuff gets reordered later.